.github/workflows/merged.yml

```diff
@@ -8,20 +8,119 @@ on:
       build_configuration:
         description: 'Which build configuration to use'
         required: false
-        default: 'Debug'
+        default: 'Staging'
         type: choice
         options:
-        - Debug
-        
+        - Staging
+        - Build
+
 permissions:
   id-token: write
   contents: read
 
 jobs:
-  build:
-    name: Build app, run Configuration and UI tests then deploy to TestFlight
+  test_staging_config:
```

Comment:
do we need to checkout the code and run linting in all 3 test steps you've got defined here?

Reply:
Update: included a separate job for linting which the test jobs depend upon before running